dev/TeachingTip/TeachingTip.h

```diff
@@ -68,8 +68,11 @@ class TeachingTip :
     winrt::Popup::Opened_revoker m_popupOpenedRevoker{};
     winrt::Popup::Closed_revoker m_popupClosedRevoker{};
     winrt::Popup::Closed_revoker m_lightDismissIndicatorPopupClosedRevoker{};
-    winrt::Window::SizeChanged_revoker m_windowSizeChangedRevoker{};
+    winrt::CoreWindow::SizeChanged_revoker m_windowSizeChangedRevoker{};
     winrt::Grid::Loaded_revoker m_tailOcclusionGridLoadedRevoker{};
+    // The XamlRoot::Changed_revoker doesn't work for unattaching the event.
+    // Tracked by internal bug #21302432.
```

Comment:
@Austin-Lamb any ideas on this? The changed_revoker is just C++/WinRT codegen that's the same for everything and it just calls revoke with an event_token itself. So I don't know why this would be different except that the revoker grabs a weak ref to the target. Maybe XamlRoot doesn't support weakrefs on it?

@StephenLPeters  did you debug through to why the revoke call wasn't revoking?

Reply:
I have no idea what's going on here.  Stephen filed a bug on us, but we haven't had a chance to look at it yet, and probably won't have time for a while.

---
In reply to: [282296441](https://github.com/microsoft/microsoft-ui-xaml/pull/568#discussion_r282296441) [](ancestors = 282296441)